index.html

```diff
@@ -66,15 +66,17 @@ <h3 class="form-label">Minutes</h3>
                   <input type="text" class="minutes-input" id="minutesInput"/>
                   <div class="category-warning warning-minutes hidden">
                     <img class="error-icon" src="./assets/warning.svg" alt="Error">
-                    <p>A number is required.</p>
+                    <p class="minutes-missing hidden">A number is required.</p>
+                    <p class="minutes-range hidden">Enter a number between 0 - 1,440.</p>
```

Comment:
Nice!! We might change it to 24 hours or something like that but we can discuss.